ci/pipelines/concourse-pypi-resource-pipeline-template.yml

```diff
@@ -0,0 +1,227 @@
+---
+docker-registry-config: &docker-registry-config
+  username: ((docker_cloud_pix4d_user))
+  password: ((docker_cloud_pix4d_password))
+
+s3-params: &s3-params
+  bucket: ci-pix4d-concourse-pipeline
+  region_name: eu-west-1
+  access_key_id: ((concourse_user_access_key))
+  secret_access_key: ((concourse_user_secret_key))
+
+resource_types:
+- name: cogito
+  type: registry-image
+  check_every: 24h
+  source:
+    repository: pix4d/cogito
+    tag: master
+
+- name: pypi.test
+  type: registry-image
+  source:
+    <<: *docker-registry-config
+    repository: docker.ci.pix4d.com/concourse-pypi-resource
+    tag: rc-((docker_tag))
+
+resources:
+- name: gchat-notify
+  type: cogito
+  icon: chat-alert
+  check_every: never
+  source:
+    sinks:
+      - gchat
+    gchat_webhook: ((pci-gchat_hook))
+
+- name: linux-builder.git
+  icon: git
+  type: git
+  source:
+    uri: git@github.com:Pix4D/linux-image-build.git
+    private_key: ((github_ssh_key))
+    branch: master
+    paths:
+      - ci/tasks/*
+
+- name: pix4d-pypi
+  type: pypi.test
+  source:
+    name: concourse.example
+    packaging: any
+    pre_release: true
+    repository:
+      authenticate: in
+      username: concourse
+      password: ((concourse_artifactory_password))
+      index_url: https://artifactory.ci.pix4d.com/artifactory/api/pypi/pix4d-pypi-dev-local/simple
+      repository_url: https://artifactory.ci.pix4d.com/artifactory/api/pypi/pix4d-pypi-dev-local
+
+- name: pypi-wheel-example
+  type: git
+  source:
+    uri: git@github.com:pix4d/concourse-examples.git
```

Comment:
Can you confirm that building (actually deploying) this example would actually fail if we use the current version of the pypi resource?

If its not, can you build an example which fails with current verison, but passes with the one now.

You can test this with your pipeline super simple if you just create a feature branch and pipeline based on this commit: https://github.com/Pix4D/concourse-pypi-resource/commit/f90b487a3fc9153f1a0c3fb9411d38a3a3a6f7dd and cherry-pick this PR.

Reply:
How do you know that it works in all cases? I am almost sure that your example would work also with old resource.
What we need is to ensure that new image works with the package that was previously failing... 
In other words, you are building a normal Python wheel which will work will all versions of Python and all versions of the resource.